transport/src/main/java/io/netty/channel/PendingWriteQueue.java

```diff
@@ -37,6 +38,11 @@
     private static final int PENDING_WRITE_OVERHEAD =
             SystemPropertyUtil.getInt("io.netty.transport.pendingWriteSizeOverhead", 64);
 
+    @UnstableApi
+    public static int pendingWriteOverhead() {
```

Comment:
do we only need to expose this for the unit test ?

Reply:
Yes, but just to reduce the likely of faiures due to wrong watermark configs: If the impl will look good will find a better way to expose this, if needed.

Reply:
Probably changing impl due to https://github.com/netty/netty/pull/9753#discussion_r342220866 I don't need to modify the test anymore and expose it TBH

Reply:
I would prefer to not expose it for now and just hard-code in the test.